libs/langgraph/langgraph/pregel/main.py

```diff
@@ -2458,6 +2515,10 @@ def stream(
                         c,
                     )
                 )
+
+            # set up custom stream mode
+            if "custom" in stream_modes:
+                config[CONF][CONFIG_KEY_STREAM_WRITER] = stream_writer
```

Comment:
We shouldn’t need this anymore, right? The existing places that read this config key can now read it from runtime instead